pom.xml

```diff
@@ -83,17 +83,17 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-annotations</artifactId>
-            <version>2.1.2</version>
+            <version>2.6.7</version>
```

Comment:
Druid is stuck on `2.4.6` due to hadoop concerns, suggest not bumping it above `2.4.6` here either